spiketools/plts/trials.py

```diff
@@ -39,15 +40,15 @@ def plot_rasters(data, vline=None, colors=None, vshade=None, show_axis=False, ax
     ax = check_ax(ax, figsize=plt_kwargs.pop('figsize', None))
 
     check = False
-    for ind in range(len(data)):
+    for val in data:
         try:
-            if isinstance(data[ind], float):
+            if isinstance(val, float):
                 break
-            elif isinstance(data[ind][0], list):
+            elif isinstance(val[0], list):
```

Comment:
I think it may be worth adding an `| isinstance(val[0], np.ndarray)` here, so that the function also handles the case in which `data` is a list of `np.arrays`. What do you think?

Reply:
yeh - that's a great idea, thanks! Added!